aea/protocols/default/__init__.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # ------------------------------------------------------------------------------
 #
-#   Copyright 2018-2020 Fetch.AI Limited
+#   Copyright 2020 fetchai
```

Comment:
Why you removed `2018`? And why `fetchai` now is lowercase?
The website uses `Fetch.ai`: https://fetch.ai/

Also, please update LICENSE: https://github.com/fetchai/agents-aea/blob/feature/protocols_updates/LICENSE#L189

Reply:
Moreover, if we change the copyright here, shouldn't we change it everywhere else?

Reply:
These are the autogenerated protocols. We cannot have a special rule for fetchai, we can in the midterm improve the generator to have a better representation.